lib/trailblazer/rails/railtie.rb

```diff
@@ -3,8 +3,37 @@
 
 module Trailblazer
   class Railtie < ::Rails::Railtie
+    def self.load_root(root)
+
+      # Loader has no concept for base dir, must chdir so that it can find files local to the root
+      Dir.chdir(root) do
+        # For STI models, we need to sort all models before concepts, gather files first from loader
+        files = []
+        Loader.new.({debug: false, insert: [ModelFile, before: Loader::AddConceptFiles]}) { |file|
```

Comment:
Shouldn't we have a new function `ModelFiles` instead of `ModelFile` that simply gathers all model files in one step, instead of loading them per concept? This would reduce logic, wouldn't it?